tagstudio/src/qt/modals/build_tag.py

```diff
@@ -51,6 +51,8 @@ def __init__(self, library: Library, tag: Tag | None = None):
         self.name_title.setText("Name")
         self.name_layout.addWidget(self.name_title)
         self.name_field = QLineEdit()
+        self.name_field.setFixedHeight(24)
+        self.name_field.textChanged.connect(self.on_name_changed)
```

Comment:
Suggestion to add some placeholder text to signal to users who don't or [can't notice](https://en.wikipedia.org/wiki/Congenital_red%E2%80%93green_color_blindness) the red outline that a name is required
```suggestion
        self.name_field.setFixedHeight(24)
        self.name_field.textChanged.connect(self.on_name_changed)
        self.name_field.setPlaceholderText("Tag Name (Required)")
```

Reply:
Seems reasonable to me.